uber/templates/emails/daily_checks/pending_emails.html

```diff
@@ -1,17 +1,28 @@
 <html>
 <head></head>
 <body>
-    <h3>{{ c.EVENT_NAME_AND_YEAR }}<br/> Emails that need approval to send:</h3>
+    <h3>{{ c.EVENT_NAME_AND_YEAR }}<br/> Emails that need approval to send on behalf of {{ primary_sender }}:</h3>
```

Comment:
Does this mean that the report sent to STOPS will claims that all unapproved emails will be sent on behalf of STOPS? Or did I read that wrong?